ocrd-tool.json

```diff
@@ -22,7 +22,7 @@
           "description": "The name of the actual binarization algorithm",
           "type": "string",
           "required": true,
-          "enum": ["sauvola", "sauvola-ms", "sauvola-ms-fg", "sauvola-ms-split", "kim", "wolf"]
+          "enum": ["sauvola", "sauvola-ms", "sauvola-ms-fg", "sauvola-ms-split", "kim", "wolf", "niblack", "singh", "otsu"]
```

Comment:
I could extend the scribo-test assets with samples for kim, wolf, niblack, singh, otsu. As they are, these are just smoke tests, comparing file sizes but for consistency's sake... Should I?

Reply:
No opinion here. But maybe we could include the self-tests of scribo into our Travis configuration.